vsintegration/src/FSharp.ProjectSystem.FSharp/Project.fs

```diff
@@ -1012,7 +1012,7 @@ namespace rec Microsoft.VisualStudio.FSharp.ProjectSystem
 
                 let stripEndingSemicolon (sb : StringBuilder) =
```

Comment:
Do you think that trimming whitespace before stripping the trailing semi-colon, or is that handled elsewhere?